recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-domd.dts

```diff
@@ -338,10 +338,22 @@
 		 <&cpg CPG_MOD 727>,
 		 <&programmable_clk0>, <&du_dotclkin1>, <&du_dotclkin2>,
 		 <&programmable_clk1>,
-		 <&cpg CPG_MOD 112>;
+		 <&cpg CPG_MOD 112>,
+		 <&cpg CPG_MOD 624>, <&cpg CPG_MOD 606>,
+		 <&cpg CPG_MOD 626>, <&cpg CPG_MOD 607>,
+		 <&cpg CPG_MOD 631>, <&cpg CPG_MOD 611>,
+		 <&cpg CPG_MOD 630>, <&cpg CPG_MOD 610>,
+		 <&cpg CPG_MOD 119>, <&cpg CPG_MOD 615>,
+		 <&cpg CPG_MOD 118>, <&cpg CPG_MOD 614>;
 	clock-names = "du.0", "du.1", "du.2", "du.3",
 				  "lvds.0", "dclkin.0", "dclkin.1",
-				  "dclkin.2", "dclkin.3", "3dge";
+				  "dclkin.2", "dclkin.3", "3dge",
+				  "vspbc", "fcpvbc",
+				  "vspbd", "fcpvbd",
+				  "vspi0", "fcpvi0",
+				  "vspi1", "fcpvi1",
+				  "fdp0", "fcpf0",
+				  "fdp1", "fcpf1";
 
```

Comment:
I haven't seen corresponding code in rcar-du driver to parse these clocks. You get names from the const array directly. Why do you need to keep clocks here?

Reply:
The correspondent code in rcar-du is located right below your gfx related code.

Reply:
I actually duplicated your code. With a minor exception that I track list of clocks both in H3 and M3 SoCs but enable only clocks available for a SoC we are running on.

Reply:
Now I see. Please treat my question as non-actual.